### ISSUE TITLE
SDK logs visible on spotlight

### ISSUE THREAD

**Issue by lucas-zimerman** [getsentry/spotlight#643]
Spotlight is a great tool, but one thing that would help me is being able to see the SDK logs on spotlight. With this feature, I'd be able to only focus on a single UI when looking for logs (plus if it has basic search filters)

**Comment by BYK** [comment]
@lucas-zimerman that is a great idea! I'll add it to the list. I see two ways to make this happen:

1. Most of our SDK's have support for capturing console/terminal messages and we can use that in combination with `debug` mode in the SDK
2. We do support custom events, so SDKs may start sending custom events to Spotlight when it is enabled.

Which one do you think is the better approach?

**Comment by lucas-zimerman** [comment]
I think option 1 works the best, it will have the console messages that will help users to see their app log and also SDK logs that help debugging the SDKs. And since we have the UI control here, there could be checkboxes to hide/show console logs/SDK logs.

**Comment by HazAT** [comment]
hmm @betegon implemented it but I don't see it in the top level menu

<img width="646" alt="Image" src="https://github.com/user-attachments/assets/d9644b87-2fe0-4d83-80bb-71678a80ef96" />

**Comment by betegon** [comment]
Not really! I thought SDK logs were solve in that PR as structured logs, but I was wrong. @BYK explanation on this https://github.com/getsentry/spotlight/pull/808#issuecomment-2922447067

**Comment by BYK** [comment]
@Shubhdeep12 I'm surprised that the SDK logging does not bypass any integration we have but this is great to know. I guess it would just be a matter of us enabling console logging integration when Spotlight is turned on?

**Comment by Shubhdeep12** [comment]
Yes consolelogging integration turned on in sentry SDK and `disableLogger : false` in next config
